recipes/force-delete-win/THIRDPARTY.yml

```diff
@@ -0,0 +1,236 @@
+---
```

Comment:
This file is probably created by `cargo-bundle-licenses`(?), can you generate that during the build instead of adding it manually here?

Reply:
Aren't supposed for licenses to be in the feedstock directly? The build only affects the redistributable packages

Reply:
That's what's being done for `pywinpty`: https://github.com/conda-forge/pywinpty-feedstock/tree/main/recipe/library_licenses

Reply:
No, licenses should be included in the resulting `conda` package. Adding `cargo-bundle-licenses --format yaml --output THIRDPARTY.yml` as part of your build should be sufficient.